### ISSUE TITLE
Consider using `force-graph` instead

### ISSUE THREAD

**Issue by sakgoyal** [donovanwest/EverythingsConnected#2]
instead of using d3 directly, you can just use `force-graph` which uses d3 under the hood and hides all the complexity for you. it has all the functionality build in so you can just delete the `graph.js` file. 
it even solves the zoom issues

**Comment by donovanwest** [comment]
Interesting. I might have to look into this sometime. Unfortunately graph.js is tightly coupled with a lot of other things going on in a way that would make some people puke so it won't be so easy to replace it. Fixing the zoom has been my white whale though that I could never fix.

**Comment by sakgoyal** [comment]
I tried doing it myself and ended up realizing that `force-graph` is canvas based. not svg based like native d3 is. so it required insane canvas code just to get a circle with dynamic text sizes like you currently have set up. I gave up after 3 hours of fighting the canvas api and it still didnt work lol. 

im resorting to taking your code and trying to modernize it with `vite`, adding `typescript`, and using the official spotify sdk. its a big change so its taking some time to fix things
